app/models/job.rb

```diff
@@ -0,0 +1,50 @@
+
+class Job < ApplicationRecord
```

Comment:
Thanks @mrGrazy ! Yeah Paul and I have discussed that as well, there is also the Sidekiq Job so it will be probably best to rename the Job model to something clearer.